etcd/test/jepsen/etcd_test.clj

```diff
@@ -24,7 +24,7 @@
                  :db        (db)
                  :client    (cas-client)
                  :model     (model/cas-register)
-                 :checker   (checker/compose {:html   timeline/html
+                 :checker   (checker/compose {;:html   timeline/html
```

Comment:
This is just a "work-around" for `java.lang.IllegalArgumentException: No implementation of method: :check of protocol: #'jepsen.checker/Checker found for class: jepsen.checker.timeline$html`